src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_button/create_button.tsx

```diff
@@ -17,8 +17,7 @@
  * under the License.
  */
 
-// @ts-ignore
-import { euiColorAccent } from '@elastic/eui/dist/eui_theme_light.json';
+import { euiTheme } from '@kbn/ui-shared-deps/eui_theme';
```

Comment:
This is an example of the types of code changes we might need to do in many parts of the code to add support for swapping out the eui_theme vars automatically. This is often done like so: https://github.com/elastic/kibana/blob/441bff86dbbb3229059496456b5611f6c6697e25/x-pack/legacy/common/eui_styled_components/eui_styled_components.tsx but all these could be migrated to use `@kbn/ui-shared-deps/eui_theme` if we think that's the right way to go. Either way, the `@elastic/eui/dist/eui_theme_dark.json` and `@elastic/eui/dist/eui_theme_light.json` imports are being automatically rewritten to their "amsterdam" equivalents.